<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectionProgessWindow</class>
 <widget class="QDialog" name="ConnectionProgessWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>199</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="stepOneLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>161</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/connection/resources/signon_blank_square.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="stepTwoLabel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>30</y>
     <width>161</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/connection/resources/signon_blank_square.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="stepThreeLabel">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>30</y>
     <width>161</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/connection/resources/signon_blank_square.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="stepOneAction">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>111</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/connection/resources/signon_step1.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="stepTwoAction">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>40</y>
     <width>111</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/connection/resources/signon_step2.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="stepThreeAction">
   <property name="geometry">
    <rect>
     <x>383</x>
     <y>40</y>
     <width>131</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/connection/resources/signon_step3.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="progressTextLabel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>150</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Step 1: Connecting to Server</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>551</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
